bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/config/CallOptionsConfig.java

```diff
@@ -223,6 +283,15 @@ public int getShortRpcTimeoutMs() {
     return shortRpcTimeoutMs;
   }
 
+  /**
+   * Getter for the field <code>shortRpcAttemptTimeoutMs</code>.
+   *
```

Comment:
also add that if null, then the attempt rpc will be bounded by the remaining operation timeout.

Reply:
Done